core/collector.class.inc.php

```diff
@@ -14,6 +14,8 @@
 //   You should have received a copy of the GNU Affero General Public License
 //   along with this application. If not, see <http://www.gnu.org/licenses/>
 
+define('NULL_VALUE', '<NULL>');
+
```

Comment:
Hi Hipska,

could you please detail the functionnal goal of this PR? what is the behaviour with current code ? what it will be after?

BR
Olivier

Reply:
This is just the definition of a constant. Similar as in iTop. If you are referring tho the whole PR, please comment at the correct place. The initial post already explains the goal.